.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: "CI"
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+jobs:
+  security_audit_and_lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
```

Comment:
you can specify the ruby-version here. Please refer this [link](https://github.com/actions/setup-ruby)

Reply:
Same comment applies to all the below use cases

Reply:
GitHub Actions using [this setup-ruby repository](https://github.com/ruby/setup-ruby). I updated to version v1 as suggested in the README, which will read the Ruby version from .ruby_version/tool_version.